src/Microsoft.Diagnostics.Monitoring.EventPipe/Counters/CounterPayload.cs

```diff
@@ -23,7 +22,6 @@ protected CounterPayload(DateTime timestamp,
         {
             Timestamp = timestamp;
             DisplayName = displayName;
-            Unit = unit;
```

Comment:
While not really necessary, leaving Unit here will make it a little easier to deal with breaking changes.

Reply:
I put it all back so dotnet-monitor builds without needing any changes.